tests/render/test_default_jinja_view.py

```diff
@@ -159,8 +160,8 @@ def test_rendering_components_without_section_loop_index():
             "content_block_loop": {"index": 2},
         }
     )
-    print(rendered_doc)
     rendered_doc = rendered_doc.replace(" ", "").replace("\t", "").replace("\n", "")
+    print(rendered_doc)
```

Comment:
Looks like you are changing the order of when these prints are being displayed.
I would defer to the original test author regarding when the debugging print statements should invoked.

Reply:
I think it would probably be easier to do a git checkout from develop but keep your marker changes.